.github/workflows/actions/configuration/action.yaml

```diff
@@ -73,4 +73,7 @@ runs:
       RAY_IMAGE="rayproject/ray:${{ inputs.ray_version }}" \
       OPERATOR_IMAGE="kuberay/operator:${{ steps.vars.outputs.sha_short }}" \
       APISERVER_IMAGE="kuberay/apiserver:${{ steps.vars.outputs.sha_short }}" python test_sample_raycluster_yamls.py
+      RAY_IMAGE="rayproject/ray:${{ inputs.ray_version }}" \
```

Comment:
KubeRay CI does not have enough CPU. Removed it from KubeRay CI.

#695 tracks the progress of integration with Ray CI.